### PR TITLE
RSDK-5103 Populate module template README.md

### PR DESCRIPTION
**Pull request by acmorrow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
...
+
+Modular resources for the Viam Micro-RDK function differently than
+modular resources for the standard Viam RDK, because they must be
+compiled into the image which will be flashed to the microcontroller.
+
+## Usage
+
+`cargo generate --git https://github.com/viamrobotics/micro-rdk-module-template`
+
+Answer the prompts, and then implement your modular resources within
```
Comment (gvaradarajan) [inline README.md:22]: maybe we should outline what prompts the user should expect?

**Review by gvaradarajan — APPROVED** [review]
LGTM, just a minor suggestion

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+
+Once you have implemented your module, you can use it in your
+Micro-RDK robot project simply by adding it as a standard dependencies
+in the `[dependencies]` section of your robot project's `Cargo.toml`
+file. The `register_models` entry point of all dependencies produced
+by this template will be automatically invoked at startup.
+
+## Tutorial
+
+Please see the [ESP32 Sensors
```
Comment (randhid) [inline README.md:37]: ah, the instructions are here - maybe a "read this doc in its entirety" right under the beta notice.

**Review by randhid — APPROVED** [review]
Few nits here as well!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,52 @@
...
+created from this template are a starting point for defining modular
+resources for the Micro-RDK.
+
+Modular resources for the Viam Micro-RDK function differently than
+modular resources for the standard Viam RDK, because they must be
+compiled into the image which will be flashed to the microcontroller.
+
+## Usage
+
+`cargo generate --git https://github.com/viamrobotics/micro-rdk-module-template`
```
[the remark this answers is not in the document]
Comment (randhid) [reply README.md:33]: section looks good thanks!

**Pull request merged**